package.json

```diff
@@ -68,7 +68,17 @@
     "generate:readme:toc": "markdown-toc -i README.md",
     "generate:assets": "npm run build && npm run docs",
     "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION",
-    "prepublishOnly": "npm run build"
+    "prepublishOnly": "npm run build",
+    "ci:blackbox": "npm run ci:blackbox:js && npm run ci:blackbox:ts && npm run ci:blackbox:java",
+    "ci:blackbox:js": "npm run ci:blackbox:js:generate && npm run ci:blackbox:js:test",
+    "ci:blackbox:js:generate": "node ./test/CI-blackbox/renderer.js js",
+    "ci:blackbox:js:test": "node ./test/CI-blackbox/js/output.js",
+    "ci:blackbox:ts": "npm run ci:blackbox:ts:generate && npm run ci:blackbox:ts:test",
+    "ci:blackbox:ts:generate": "node ./test/CI-blackbox/renderer.js ts",
+    "ci:blackbox:ts:test": "tsc -t es5 ./test/CI-blackbox/ts/output.ts && node ./test/CI-blackbox/ts/output.js",
+    "ci:blackbox:java": "npm run ci:blackbox:java:generate && npm run ci:blackbox:java:test",
+    "ci:blackbox:java:generate": "node ./test/CI-blackbox/renderer.js java",
+    "ci:blackbox:java:test": "javac ./test/CI-blackbox/java/*.java"
```

Comment:
Do we really need these scripts? If we add another language then what, new scripts? 😅 Maybe we should use this trick with `npm run blackbox -- {languages}`?

Reply:
Really didnt want to add my own sub-process spawn in the renderer... What if we refactored it to 
```
    "ci:blackbox:generator": "node ./test/CI-blackbox/renderer.js",
    "ci:blackbox:ts:test": "tsc -t es5 ./test/CI-blackbox/ts/output.ts && node ./test/CI-blackbox/ts/output.js",
    "ci:blackbox:java:test": "javac ./test/CI-blackbox/java/*.java"
```

Reply:
It's better that previous, but as Łukasz wrote, maybe we should switch to Jest runner? Probably it will simplify scripts only for TS and JS models, not for Java, but maybe... what do you think? 😄